go/vt/vttest/local_cluster.go

```diff
@@ -151,20 +151,20 @@ type Config struct {
 // It then sets the right value for cfg.SchemaDir.
 // At the end of the test, the caller should os.RemoveAll(cfg.SchemaDir).
 func (cfg *Config) InitSchemas(keyspace, schema string, vschema *vschemapb.Keyspace) error {
-	if cfg.SchemaDir != "" {
-		return fmt.Errorf("SchemaDir is already set to %v", cfg.SchemaDir)
-	}
-
-	// Create a base temporary directory.
-	tempSchemaDir, err := os.MkdirTemp("", "vttest")
-	if err != nil {
-		return err
+	schemaDir := cfg.SchemaDir
+	if schemaDir == "" {
+		// Create a base temporary directory.
+		tempSchemaDir, err := os.MkdirTemp("", "vttest")
+		if err != nil {
+			return err
+		}
+		schemaDir = tempSchemaDir
```

Comment:
The main change in this file is the above lines. 
The below is a trivial diff.

Reply:
OK. WHY are we doing this? 🙂